### DISCUSSION TITLE
Account not found / DEX

### DISCUSSION THREAD

**Discussion by RomanAnikienko** [trustwallet/wallet-core#1287] [Q&A]
Recently, I decided to use DEX in Trust Wallet, but found one problem...
When I try to open an order in TrustWallet via DEX, I get the message "Account not found". (Screenshot attached)

Dear developers, how to solve this issue?
![2021-02-06 14 08 41](https://user-images.githubusercontent.com/953473/107112861-017f6f80-6885-11eb-9f12-1237f763f4e3.jpg)

**Comment by hewigovens** [comment]
This will be fixed in next release
